diff -c a/ports/tbb/CONTROL b/ports/tbb/CONTROL
*** a/ports/tbb/CONTROL Wed Jan 13 19:01:14 2021
--- b/ports/tbb/CONTROL Wed Jan 13 19:04:45 2021
***************
*** 2,5 ****
  Version: 2020_U3
  Homepage: https://github.com/01org/tbb
  Description: Intel's Threading Building Blocks.
! Supports: !(uwp|arm|arm64)
\ No newline at end of file
--- 2,5 ----
  Version: 2020_U3
  Homepage: https://github.com/01org/tbb
  Description: Intel's Threading Building Blocks.
! Supports: !(uwp)
diff -c a/ports/tbb/portfile.cmake b/ports/tbb/portfile.cmake
*** a/ports/tbb/portfile.cmake  Wed Jan 13 19:01:14 2021
--- b/ports/tbb/portfile.cmake  Wed Jan 13 19:04:48 2021
***************
*** 1,5 ****
- vcpkg_fail_port_install(ON_ARCH "arm" "arm64" ON_TARGET "uwp")
-
  vcpkg_from_github(
      OUT_SOURCE_PATH SOURCE_PATH
      REPO oneapi-src/oneTBB
--- 1,3 ----
